infrastructure/smart-contracts/contracts/upgrades/referee/Referee9.sol

```diff
@@ -1112,16 +1062,51 @@ contract Referee9 is Initializable, AccessControlEnumerableUpgradeable {
     }
 
     /**
-    * @notice Claim Pool Submission Rewards external function
-    * @dev this function is called by the pool owner, or an approved operator, to claim rewards for a pool submission.
-    * @param _poolAddress The address of the pool.
+    * @notice Claim Bulk rewards for either a pool, or a wallet owner with unstaked keys.
+    * @dev this function is called by the pool owner, an approved operator, or a sentry wallet owner to claim rewards for a bulk submissions.
+    * @param _bulkAddress The address of the pool or owner wallet.
     * @param _challengeId The ID of the challenge.
     */
-    function claimPoolSubmissionRewards(address _poolAddress, uint256 _challengeId) external {
+    function claimBulkRewards(address _bulkAddress, uint256 _challengeId) external {
+
```

Comment:
Might be a good chance to make this function allow multiple bulkaddresses to claim for within one transaction, could make our life in the operator easier, let me know what you think

Reply:
I agree, should submissions also work the same way? An array of addresses?